crypto/keyring/keyring_test.go

```diff
@@ -1211,32 +1360,35 @@ func TestRenameKey(t *testing.T) {
 	}
 }
 
-func requireEqualRenamedKey(t *testing.T, newKey, oldKey Info) {
-	require.NotEqual(t, newKey.GetName(), oldKey.GetName())
-	require.Equal(t, newKey.GetAddress(), oldKey.GetAddress())
-	require.Equal(t, newKey.GetPubKey(), oldKey.GetPubKey())
-	require.Equal(t, newKey.GetAlgo(), oldKey.GetAlgo())
-	require.Equal(t, newKey.GetType(), oldKey.GetType())
-}
+func requireEqualRenamedKey(t *testing.T, key *Record, mnemonic *Record, nameMatch bool) {
+	if nameMatch {
```

Comment:
I added 'nameMatch'  argument to ` requireEqualRenamedKey` as in some cases i dont need name check ( e.g. at L1319). If it is set to true, it  checks if `key.Name` and `mnemonic.Name` match as you can see below. I hope it sounds good to you, @robert-zaremba